ui/page/send/page.go

```diff
@@ -96,23 +99,35 @@ type selectedUTXOsInfo struct {
 	totalUTXOsAmount int64
 }
 
-func NewSendPage(l *load.Load) *Page {
+func NewSendPage(l *load.Load, isModalLayout bool) *Page {
 	pg := &Page{
-		Load:             l,
-		GenericPageModal: app.NewGenericPageModal(SendPageID),
-		sendDestination:  newSendDestination(l),
-		amount:           newSendAmount(l),
-
-		exchangeRate: -1,
+		Load: l,
```

Comment:
What inside `Load` does this page need? If it's just theme, then change `Load` to theme please.

Reply:
Load has information on the wallet loader that is used in multiple places for setting the asset Manger details. Plus updating this weill most likely affect many files.

example use case is `pg.WL.AssetsManager.GetCurrencyConversionExchange()` and functions needing `pg.WL`